editor/src/utils/utils.test-utils.ts

```diff
@@ -423,14 +423,29 @@ export function slightlyOffsetPointBecauseVeryWeirdIssue(point: { x: number; y:
   return { x: point.x - 0.001, y: point.y - 0.001 }
 }
 
-export async function expectSingleUndoStep(
+async function expectNUndoSteps(
```

Comment:
Probably worth exporting this one as well

Reply:
I didn't export it on purpose because I think expecting 0 and 1 undo steps are the only sensible things to assert on, because
- you need to assert that 0 undo steps are generated if you want to assert that a disabled UI element doesn't generate an undo step even if it's clicked on
- you need to assert that 1 undo step is generated using a control (this was the original helper).
I think asserting that multiple undo steps are generated is an antipattern. Should the need for this arise while writing tests, either the test should be refactored so that a single UI interaction is wrapped in a single `expectSingleUndoStep` (or `expectNoAction`), or it the assert shouldn't be there at all because it asserts something that we consider a bug (having multiple undo steps for a single UI action)

Reply:
I guess the argument for premature optimization can go both ways - to me this sounds like it and I don't see testing for `N` steps as an anti-pattern per se, depending on the use cases; hence my suggestion since we'd be exporting the general purpose method _and_ the special cases with "nice names" (of course the counter argument stands, which is that only the "relevant" functions are exported and the implementation is hidden, if there's really no point in testing `> 1` steps). But no point in bikeshedding on this I guess, my review approval stands :)